content/applications/websites/website/publish/multi_website.rst

```diff
@@ -126,6 +126,14 @@ be redirected to the right website using GeoIP.
 .. seealso::
    - :doc:`/administration/maintain/domain_names`
 
+If you setup an extra website and specify the domain name you will have to
+do three things to get this feature to work:
+1. Setup an A-record in the DNS configuration of your hosting provider so
+that "www.mywebsite2.ext" is redirected to the IP of your Odoo instance.
+2. Create an extra Nginx ``server`` block in which you set the
+``server_name`` as you've entered in the "Website domain" field.
```

Comment:
Since `Website domain` is an Odoo field name, it could be flagged with the `guilabel` RST role.
```suggestion
``server_name`` that you've set in the :guilabel:`Website domain` field.
```